### DISCUSSION TITLE
Collecting data from Cumulus Linux MIBS - help

### DISCUSSION THREAD

**Discussion by gavmckee80** [prometheus/snmp_exporter#960] [Q&A]
Hi ,

Would it be possible to get some help to collect data from the following MIBS, I'm struggling with the generation of a configuration that can export this data.

https://docs.nvidia.com/networking-ethernet-software/mibs/CUMULUS-RESOURCE-QUERY-MIB.txt
https://docs.nvidia.com/networking-ethernet-software/mibs/CUMULUS-COUNTERS-MIB.txt
https://docs.nvidia.com/networking-ethernet-software/mibs/CUMULUS-BGPUN-MIB.txt

https://docs.nvidia.com/networking-ethernet-software/cumulus-linux-55/Monitoring-and-Troubleshooting/Simple-Network-Management-Protocol-SNMP/Supported-MIBs/

Would be great to have support for these merged into the main repo.

Thanks

Gav

**Comment by SuperQ** [comment]
For Cumulus, it's probably better to use a native exporter, for example [cumulus_exporter](https://github.com/tynany/cumulus_exporter).
